File: lua/refactoring/tests/refactor/123/java/multiple-vars/inline_var.expected.java
class Order {
  int quantity;
  int itemPrice;
}

class Inline {
  double orderCalculation(Order order) {
    var i = 3.0f;

    return order.quantity * order.itemPrice -
        Math.max(0, order.quantity - 500) * order.itemPrice * 0.05 +
        Math.min(order.quantity * order.itemPrice * 0.1, 100.0);
  }
}

File: lua/refactoring/tests/refactor/123/csharp/multiple-vars/inline_var.expected.cs
namespace Test {
    class Order
    {
        int quantity;
        int itemPrice;
    }

    class Inline
    {
        double orderCalculation(Order order)
        {
            var i = 3.0f;

            return order.quantity * order.itemPrice -
                Math.Max(0, order.quantity - 500) * order.itemPrice * 0.05 +
                Math.Min(order.quantity * order.itemPrice * 0.1, 100.0);
        }
    }
}

